.circleci/config.yml

```diff
@@ -20,17 +20,17 @@ workflows:
           context: Non-sensitive Env Vars
           requires:
             - build
-      - gh/release:
-          context: Open Source Release Credentials
-          filters:
-            branches:
-              only: master
-          requires:
-            - predeploy
-          title: Circle CI release
-          tag: v$MVN_VERSION
-          files: target/ebx-linkedin-sdk-${MVN_VERSION}*.jar
-          notes-file: CHANGELOG.md
+      # - gh/release:
```

Comment:
should like just be removed? we can always retrieve it from git history if we want it back

just to avoid commented out code in the repo

Reply:
Sure, happy to remove it. It might even be better to have the gh/release step handled by a GitHub action rather than Circle CI anyway come to think of it...